src/Servers/Kestrel/Core/src/Internal/Http2/Http2Connection.cs

```diff
@@ -786,7 +785,7 @@ private Task ProcessPingFrameAsync(in ReadOnlySequence<byte> payload)
                 return Task.CompletedTask;
             }
 
-            return _frameWriter.WritePingAsync(Http2PingFrameFlags.ACK, payload).GetAsTask();
+            return _frameWriter.WritePingAsync(Http2PingFrameFlags.ACK, payload).AsTask();
```

Comment:
Any reason for preferring AsTask() to GetAsTask()?

Reply:
I recently changed these to GetAsTask. This will be a merge issue